README.md

```diff
@@ -30,11 +30,15 @@ Before proceeding, make sure that there are no current builds in the `Build` ste
 
 6. Once the antora.yml [`staging`](https://github.com/OpenLiberty/docs/blob/staging/antora.yml) and [`draft`](https://github.com/OpenLiberty/docs/blob/draft/antora.yml) versions have been updated in the docs repo, create a pull request from `staging` to `vNext` to update the `antora.yml` version in `vNext`.
 
-7. (Temporary for now). Create a branch off of the `prod branch` of the [openliberty.io repo](https://github.com/OpenLiberty/openliberty.io). Make the following updates all in the same branch:
-   * Until Antora natively supports 'latest' redirecting to the latest version, we have to update what version `latest` in the url redirects to: Update the redirects for `/docs/latest/*`, `/docs/latest/`, and `/docs/latest` in the [redirect file](https://github.com/OpenLiberty/openliberty.io/blob/staging/src/main/webapp/WEB-INF/redirects.properties) to redirect to the new released doc version (without a `v` in the url).
+7. Repeat steps 5 and 6 for the [docs-generated repo](https://github.com/OpenLiberty/docs-generated/blob/vNext/antora.yml)
+
+8. Until Antora natively supports 'latest' redirecting to the latest version, we have to update what version `latest` in the url redirects to. In this repo, update [the redirects file on `prod`](https://github.com/OpenLiberty/docs-playbook/blob/prod/doc-redirects.properties) so that `/docs/latest/*`, `/docs/latest/`, and `/docs/latest` redirect to the new released doc version (without a `v` in the url). Then, update the same file on [`draft`](https://github.com/OpenLiberty/docs-playbook/blob/draft/doc-redirects.properties) and [`staging`](https://github.com/OpenLiberty/docs-playbook/blob/staging/doc-redirects.properties) to point to the _next_ upcoming release (the newest releaed version + 1).
```

Comment:
There's one minor typo here of "releaed"

Reply:
fixed, thanks for catching